src/main/scala/io/shiftleft/fuzzyc2cpg/cfg/AstToCfgConverter.scala

```diff
@@ -50,6 +64,16 @@ class AstToCfgConverter[NodeType](entryNode: NodeType, exitNode: NodeType, adapt
     }
   }
 
+  private implicit class EdgeBuilderWrapper2(edgeBuilder: EdgeBuilderType) {
```

Comment:
Can we not have classes called `$foo2`? Do we still need EdgeBuilderWrapper[1]?

Reply:
Done.